docs-kits/kits/Eco_Pass_KIT/changelog.md

```diff
@@ -34,19 +45,19 @@ All notable changes to this Kit will be documented in this file.
 - Added detailed data retrieval flow
 - Added Battery Pass Success Story
 
-### Updated
+### 24.03 Updated
 
 - Updated battery pass model to latest version `v4.0.0`
 - Updated Business Architecture Diagram
 
 ## [0.1.0] - 2023-08-11
 
-### Added
+### 23.08 Added
```

Comment:
Please change to 23.09 - 23.08 does not exist

Reply:
Done